### PR TITLE
Add PSData to module manifest; add a warning to remind user to run 'gcloud init' after installing the Cloud SDK

### PR DESCRIPTION
**Pull request by quoctruong**
@chrsmith Do we have a URL to an icon for the module?

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/478)

### CODE REVIEW

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 [hunk 1]
```diff
@@ -92,20 +92,29 @@ PrivateData = @{
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        # Tags = @()
+        Tags = @('Google', 'Cloud')
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: Should we add more tags? BigQuery? Cloud SQL? ect?

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 [hunk 2]
```diff
@@ -92,20 +92,29 @@ PrivateData = @{
...
-        # ProjectUri = ''
+        ProjectUri = 'https://github.com/GoogleCloudPlatform/google-cloud-powershell'
 
         # A URL to an icon representing this module.
         # IconUri = ''
 
         # ReleaseNotes of this module
-        # ReleaseNotes = ''
+        ReleaseNotes = @"
+    This module contains a set of cmdlets that lets you manage Google Cloud Platform resources.
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: nit: let you

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 [hunk 3]
```diff
@@ -92,20 +92,29 @@ PrivateData = @{
...
+    Currently, the supported resources are Google Cloud Storage resources, Google Compute Engine
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: I think you should drop the "resources" after each item.  It seems a little redundant.

File: Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psd1 [hunk 4]
```diff
@@ -96,20 +96,29 @@ PrivateData = @{
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        # Tags = @()
+        Tags = @('Google', 'Cloud')
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psd1]: Same comments on this file

File: Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psm1 [hunk 5]
```diff
@@ -93,6 +93,7 @@ function Install-GCloudSdk {
                 Write-Warning "You may have to restart the shell before gcloud can be used."
             }
         }
+        Write-Warning "Please also make sure to run 'gcloud init' before using the module."
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psm1]: This change seems out of place.  Can you update the PR description?

**Comment by quoctruong** [comment]
@iantalarico PTAL

**Pull request merged**